puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
I rather mean, "why do you need empty object at all" Couldn't `undefined` mean exactly same thing as `{}`?